variable.h

```diff
@@ -72,6 +72,7 @@ extern char **environ;
 #define VAR_YASH_AFTER_CD             "YASH_AFTER_CD"
 #define VAR_YASH_LE_TIMEOUT           "YASH_LE_TIMEOUT"
 #define VAR_YASH_LOADPATH             "YASH_LOADPATH"
+#define VAR_YASH_RC                   "YASH_RC"
```

Comment:
Do you think this variable name sufficient or is there another naming scheme you had in mind?

Reply:
Does Yash define any help functions to concatenate paths? I can't seem to find any definitions.

Reply:
No, but we have a custom string buffer implementation for generic string manipulation.

[Here](https://github.com/magicant/yash/blob/ac33b94b8a57555304fe26410bca9861872185dc/yash.c#L312-L321) is an example of how to use the buffer. See strbuf.c and strbuf.h for the available functions.